scripts/lotus-daemon.service

```
@@ -11,6 +11,8 @@ Restart=always
 RestartSec=10
 
 MemoryAccounting=true
+# Depending on the complexity of API usage of this node, you may need to adjust the following
+# memory options. Please monitor the node's performance and adjust accordingly.
 MemoryHigh=8G
 MemoryMax=10G
```

```suggestion
MemoryHigh=16G
MemoryMax=20G
```

:shrug: my node was struggling to even stay in sync so it would be good to lift this at bit. Suggestions? 12/16 maybe as a compromise if we're still aiming for laptop installs?

---

Even just having a `MemoryHigh` is probably causing heavy performance degradation aside from the memory limitation since the process doesn't get to know this pressure is being applied except that allocation calls are slower and reclamation efforts are more aggressive. Ideally this would be removed entirely but it does get very hungry on some of the big API calls.